README.md

```diff
@@ -1 +1,9 @@
-# Tic-Tac-Toe
\ No newline at end of file
+# Tic-Tac-Toe
+
+This is the third project of the Main Ruby curriculum at Microverse - @microverseinc
+
+The goal was to build a simple Tic-Tac-Toe game that runs from the command line, using Ruby
+
+The project was completed using Ruby. [Assignment link] https://www.theodinproject.com/courses/ruby-programming/lessons/oop#project-1-tic-tac-toe   (Project #1)
```

Comment:
You do not necessarily have reference Odin Project. Makes it seem for an assignment like rather than a project for your portfolio. You don't have to reference the Odin project, you can simply use the description of the project from the page, summarize it and add it to your README so that person viewing project may understand it. [OPTIONAL]